src/pages/dashboard/dashboard.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, ViewChild } from '@angular/core';
+import { IonicPage, NavController, NavParams, Platform, Navbar, AlertController } from 'ionic-angular';
+import { AuthenticationProvider } from '../../providers/authentication/authentication';
+import { PracticeableBasePageComponent } from '../../shared/classes/practiceable-base-page';
+import { Store, select } from '@ngrx/store';
+import { StoreModel } from '../../shared/models/store.model';
+import { DashboardViewDidEnter } from './dashboard.actions';
+import { Observable } from 'rxjs/Observable';
+import { DeviceAuthenticationProvider } from '../../providers/device-authentication/device-authentication';
+import { Subscription } from 'rxjs/Subscription';
+import { AppConfigProvider } from '../../providers/app-config/app-config';
+import { map } from 'rxjs/operators';
+import { getAppInfoState } from '../../modules/app-info/app-info.reducer';
+import { getVersionNumber } from '../../modules/app-info/app-info.selector';
+import { DateTimeProvider } from '../../providers/date-time/date-time';
+import { DateTime } from '../../shared/helpers/date-time';
+import { DeviceProvider } from '../../providers/device/device';
+import { Insomnia } from '@ionic-native/insomnia';
+import { ScreenOrientation } from '@ionic-native/screen-orientation';
+import { ExaminerRoleDescription } from '../../providers/app-config/constants/examiner-role.constants';
+
+interface DashboardPageState {
+  appVersion$: Observable<string>;
+  todaysDate$: Date | DateTime | string;
```

Comment:
This isn't an observable. Could be easily just a class level property, which is just initialised once.

Reply:
Agreed.

Reply:
There are also some unused properties on the class that can be removed too